src/modules/support/events/channelCreate.ts

```diff
@@ -0,0 +1,26 @@
+import { GuildChannel } from "discord.js";
+
+import channels from "../../../channels";
+import { pmdDB } from "../../../database/client";
+import { Ticket } from "../classes/Ticket";
+
+const coll = pmdDB.collection("userSettings");
+
+module.exports = async (channel: GuildChannel) => {
+	if (channel.parentID === channels.ticketCategory) {
+		const users = await coll.find({ seeAllTickets: true }).toArray();
+		users.map(async user => {
+			const ticket = new Ticket(),
+				ticketFound = await ticket.fetch("channel", channel.id);
```

Comment:
just wondering if that is correct :/

Reply:
it is, feel free to find same thing in command seealltickets :)